test/build.info

```diff
@@ -260,7 +260,7 @@ IF[{- !$disabled{tests} -}]
 
   INCLUDE[testutil.o]=..
   INCLUDE[ssl_test_ctx.o]=../include
-  INCLUDE[handshake_helper.o]=../include
+  INCLUDE[handshake_helper.o]=../include ../ssl
```

Comment:
Okay, pushed an update that replaces the PACKET usage with unrolled accesses (that are twice as long).